Makefile.dep

```diff
@@ -555,3 +530,13 @@ ifneq (,$(filter random,$(USEMODULE)))
         USEMODULE += tinymt32
     endif
 endif
+
+# include package dependencies
+-include $(USEPKG:%=$(RIOTPKG)/%/Makefile.dep)
```

Comment:
Yep. lwip has it's dependencies still in the main Makefile.dep, that's why no explicit "USEPKG" is needed.